### PR TITLE
update branch to main

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -52,7 +52,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1d
```
Comment (cachedout) [inline .ci/jobs/defaults.yml:55]: Seems unrelated to this change. Move to another PR?

File: README.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-
-## Contributing
-
-See [contributing documentation](CONTRIBUTING.md).
-
-## License
-This software is licensed under the [Apache 2 license](https://github.com/elastic/ecs-logging-php/blob/master/LICENSE).
+[![Build Status](https://apm-ci.elastic.co/buildStatus/icon?job=apm-agent-php%2Fecs-logging-php-mbp%2Fmain)](https://apm-ci.elastic.co/job/apm-agent-php/job/ecs-logging-php-mbp/job/main/)
+
+# ECS Logging for PHP
```
Comment (cachedout) [inline README.md:3]: How did all this change? Did a formatter kick in?

Comment (pimjansen) [reply README.md:3]: Isnt it a line ending change? CRLF / LF

File: .ci/jobs/defaults.yml [hunk 3]
```diff
@@ -52,7 +52,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
```
Comment (v1v) [inline .ci/jobs/defaults.yml:54]:
```suggestion
        wipe-workspace: 'True'
    periodic-folder-trigger: 1w
```

File: README.md (hunk 2 above)
Comment (v1v) [reply README.md:3]: I was not expecting this, so I'll see if I can revert it

File: .ci/jobs/defaults.yml [hunk 4]
```diff
@@ -52,7 +52,7 @@
...
-    periodic-folder-trigger: 1d
+    periodic-folder-trigger: 1w
```
Comment (v1v) [inline .ci/jobs/defaults.yml:55]:
```suggestion
    periodic-folder-trigger: 1d
```

**Review by cachedout — APPROVED** [review]
Thanks for cleaning this one up. :)
